blackboard/gui/result_pane.py

```diff
@@ -0,0 +1,30 @@
+import gi
+gi.require_version('Gtk', '3.0')
+
+from gi.repository import Gtk, Gio, Gdk  # noqa
+
+
+class result_pane(Gtk.Grid):
+    def __init__(self):
+        Gtk.Grid.__init__(self)
+        self.__init_lb_list()
+
+    def __init_lb_list(self):
+        lb = Gtk.Label(str(0))
+        self.lb_list = []
+        self.lb_list.append(lb)
+        self.attach(lb, 0, 0, 1, 1)
+        lb.show()
+
+    def add_label(self, txt_op, value):
+        if Gdk.keyval_name(value.keyval) == 'Return':
+            x = txt_op.get_cursor_locations()
```

Comment:
Yes!
I've moved this to a method call "capture_pressed_key". Now, this method calls "__add_label" method.
See commit d3bd51a2b6ae4e9d234e16c2c0e3ad36a6aa3ecf